docs/create-packages/Symbol-Packages-V2.md

```diff
@@ -0,0 +1,101 @@
+
+Example commands to create .snupkg files
+```
+dotnet.exe pack MyPackage.csproj --include-symbols -p:SymbolPackageFormat=snupkg
+
+nuget.exe pack MyPackage.nuspec -Symbols -SymbolPackageFormat snupkg
+
+nuget.exe pack MyPackage.csproj -Symbols -SymbolPackageFormat snupkg
+```
+
+## Symbol package structure
+
+The .nupkg file would be exactly the same as it is today, but the .snupkg file would have the following characteristics:
+
+1) The .snupkg will have the same id and version as the corresponding .nupkg.
+2) The .snupkg will have the exact folder structure as the nupkg for any DLL or EXE files with the distinction that instead of DLLs/EXEs, their corresponding PDBs will be included in the same folder hierarchy. Files and folders with extensions other than PDB will be left out of the snupkg.
+3) The .nuspec file in the .snupkg will also specify a new PackageType as follows:
+``` 
+<packageTypes>
+  <packageType name="SybmolsPackage"/>
+</packageTypes>
+```
+4) If an author decides to use a custom nuspec to build their nupkg and snupkg, the snupkg should have the same folder hierarchy and files detailed in 2).
+5) ```authors``` and ```owners``` field will be excluded from the snupkg's nuspec.
+
+
+## Nuget.org symbol package constraints
+The symbol packages supported on nuget.org have the following contraints
+
+- Only the following file extensions are allowed to be added to a symbol package. ```.pdb,.nuspec,.xml,.psmdcp,.rels,.p7s```
+- Only managed [Portable pdbs](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md) are currently supported on nuget symbol server.
+- The pdbs and associated nupkg dlls need to be built with the compiler in Visual Studio version 15.9 or above (see [pdb crypto hash](https://github.com/dotnet/roslyn/issues/24429))
+
+The symbol package publish on nuget.org will fail if any other file types are included in the .snupkg.
+
+## Publishing a symbol package
+
+1. For convenience, first save your API key with NuGet (see [publish a package](../create-packages/publish-a-package.md)).
+
+    ```cli
+    nuget SetApiKey Your-API-Key
+    ```
+
+1. After publishing your primary package to nuget.org, push the symbol package as follows.
+
+    ```cli
+    nuget push MyPackage.snupkg
+    ```
+
+1. You can also push both primary and symbol packages at the same time using the below command. Both, .nupkg and .snupkg files need to be present in the current folder.
+
+    ```cli
+    nuget push MyPackage.nupkg
+    ```
+
+In this case, NuGet will publish to nuget.org the `MyPackage.nupkg` first followed by `MyPackage.snupkg`.
+
+## Symbol package validation and indexing
+Symbol packages published to [NuGet.org](https://www.nuget.org/) undergo several validations, such as virus checks.
```

Comment:
are we advertising that we do virus checks?

Reply:
I do not think that there is anything wrong in mentioning the virus checks. We contact the users when there is a virus reported.

Reply:
Also this text was copied from similar text for package validation documentation. So, we already do mention it.